### PR TITLE
Cluster dev env: only run migrations on first node

### PR DESCRIPTION
**Pull request by shanemcd**

### CODE REVIEW

File: tools/ansible/roles/dockerfile/files/wait-for-migrations [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+wait_for() {
+    local rc=1
+    local attempt=1
+    local next_sleep="${MIN_SLEEP}"
+
+    while true; do
+        log_message "Attempt ${attempt} of ${ATTEMPTS}"
+
+        timeout "${TIMEOUT}" \
+                /bin/bash -c "! awx-manage showmigrations | grep '\[ \]'" &>/dev/null \
```
Comment (ryanpetrello) [inline tools/ansible/roles/dockerfile/files/wait-for-migrations:30]: `awx-manage check_migrations --dry-run --check` might be a cleaner way to get an exit code without having to search for text

Comment (shanemcd) [reply tools/ansible/roles/dockerfile/files/wait-for-migrations:30]: That command seems to be broken

```
bash-4.4$ awx-manage check_migrations --dry-run --check
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 217, in ensure_connection
    self.connect()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 195, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 194, in get_new_connection
    conn = Database.connect(**conn_params)
sqlite3.OperationalError: unable to open database file
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/local/bin/awx-manage", line 9, in <module>
    load_entry_point('awx', 'console_scripts', 'awx-manage')()
  File "/awx_devel/awx/__init__.py", line 164, in manage
    execute_from_command_line(sys.argv)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/awx_devel/awx/main/management/commands/check_migrations.py", line 13, in execute
    return MakeMigrations().execute(*args, **options)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/core/management/commands/makemigrations.py", line 101, in handle
    loader.check_consistent_history(connection)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/loader.py", line 283, in check_consistent_history
    applied = recorder.applied_migrations()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/recorder.py", line 73, in applied_migrations
    if self.has_table():
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/migrations/recorder.py", line 56, in has_table
    return self.Migration._meta.db_table in self.connection.introspection.table_names(self.connection.cursor())
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 256, in cursor
    return self._cursor()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 233, in _cursor
    self.ensure_connection()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 217, in ensure_connection
    self.connect()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 217, in ensure_connection
    self.connect()
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/base/base.py", line 195, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/django/db/backends/sqlite3/base.py", line 194, in get_new_connection
    conn = Database.connect(**conn_params)
django.db.utils.OperationalError: unable to open database file
```

Comment (shanemcd) [reply tools/ansible/roles/dockerfile/files/wait-for-migrations:30]: Looking into it a bit more, it doesn't seem like this command could have ever worked in the dev env. And that concludes the amount of time I want to spend on this 😄

**Review by ryanpetrello — APPROVED** [review]

**Comment by jakemcdermott** [comment]
regate

**Pull request merged**